bgpd/bgp_table.h

```diff
@@ -52,6 +53,10 @@ struct bgp_node {
 
 	struct bgp_node *prn;
 
+	STAILQ_ENTRY(bgp_node) pq;
+	afi_t afi;
```

Comment:
you can retrieve afi & safi from node->table => would be better to not duplicate these fields here